python-frontend/src/test/java/org/sonar/python/PythonVisitorCheckTest.java

```diff
@@ -127,7 +130,27 @@ public void initialize(SubscriptionCheck.Context context) {
     SubscriptionVisitor.analyze(Collections.singletonList(check), context);
   }
 
+  @Test
+  public void stubFilesSymbols() throws IOException {
+    Collection<Symbol> symbols = TypeShed.stubFilesSymbols();
+    PythonVisitorContext context = TestPythonVisitorRunner.createContext(FILE);
+
+    SymbolsRecordingCheck check = new SymbolsRecordingCheck();
+    check.scanFile(context);
+    SubscriptionVisitor.analyze(Collections.singletonList(check), context);
+
+    assertThat(check.symbols).isEqualTo(TypeShed.stubFilesSymbols());
```

Comment:
I guess you could use `symbols` assigned earlier. Or otherwise remove the variable altogether as it is not used.

Reply:
Fixed (removed the variable completely), but rewrote history as was asked earlier to keep history clean.